digiwf-libs/digiwf-camunda-prometheus/digiwf-camunda-prometheus-core/src/main/java/de/muenchen/oss/digiwf/camunda/prometheus/ExecutionEventReporter.java

```diff
@@ -0,0 +1,54 @@
+package de.muenchen.oss.digiwf.camunda.prometheus;
+
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Counter;
+import lombok.RequiredArgsConstructor;
+import org.camunda.bpm.engine.RepositoryService;
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.springframework.context.event.EventListener;
+
+@RequiredArgsConstructor
+public class ExecutionEventReporter implements MetricsReporter {
+
+    private final RepositoryService repositoryService;
+    private Counter startExecutionEvents;
+    private Counter endExecutionEvents;
+
+    @Override
+    public void registerMetrics(CollectorRegistry collectorRegistry) {
+        this.startExecutionEvents = Counter.build()
+            .name("camunda_start_process_events_count")
+            .help("Start process events by process definition.")
+            .labelNames("processDefinitionKey")
+            .register(collectorRegistry);
+        this.endExecutionEvents = Counter.build()
+            .name("camunda_end_process_events_count")
+            .help("End process events by process definition.")
+            .labelNames("processDefinitionKey")
+            .register(collectorRegistry);
+    }
+
+    @EventListener(condition = "#execution.eventName.equals('start')")
```

Comment:
@zambrovski fliegt das event wirklich nur 1 Mal pro Definition?

Reply:
Nein. Pro start der Execution.

Reply:
Was ist mit StartEvents in Subprozessen? Sollte das auf die Root InstanceId eingeschränkt werden?